packages/framework/deprecated-agent-scheduler-container-runtime/src/test/test.spec.ts

```diff
@@ -0,0 +1,7 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// Need some test to generate eslint log correctly
+describe.skip("Deprecated functionality", () => { });
```

Comment:
i don't understand this.

Reply:
Eslint requires some file present here to complete successfully -- I'll remove the empty skipped test though, it works fine with an empty file.